web/src/beta/lib/reearth-ui/components/Modal/index.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { FC, useCallback, useState } from "react";
+
+import { Button } from "../Button";
+import { ModalPanel } from "../ModalPanel";
+
+import { ModalProps, Modal as ModalComponent } from ".";
+
+const meta: Meta<ModalProps> = {
+  component: ModalComponent,
+};
+
+export default meta;
+type Story = StoryObj<typeof ModalComponent>;
+
+const Modal: FC<ModalProps & { size: "small" | "medium" | "large"; onClose?: () => void }> = ({
+  size,
+  children,
+}) => {
+  const [visible, setVisible] = useState(false);
+
+  const handleOpen = useCallback(() => {
+    setVisible(true);
+  }, []);
+
+  const handleClose = useCallback(() => {
+    setVisible(false);
+  }, []);
+
+  return (
+    <div style={{ height: "50vh" }}>
+      <Button title="Open Modal" appearance="primary" onClick={handleOpen} />
+      {visible && (
+        <ModalComponent size={size} visible={visible}>
```

Comment:
```suggestion
        <ModalComponent size={size} visible={visible} onClose={handleClose}>
```
You added `escape to close` feature, but didn't passing in the handle function.